apps/reactlit-examples/src/components/theme-toggle.tsx

```diff
@@ -1,10 +1,21 @@
 'use client';
+import { useEffect, useState } from 'react';
 import { Button, Tooltip } from '@radix-ui/themes';
 import { MoonIcon, SunIcon } from 'lucide-react';
 import { useTheme } from 'next-themes';
 
 export function ThemeToggle() {
   const { resolvedTheme, setTheme } = useTheme();
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
```

Comment:
setMounted can be a dependency here

Reply:
setMounted is stable across re-renders. It won't change throughout the lifecycle of the component. I think we don't need to add it as dependency

Reply:
I'll fix this in a subsequent commit, usually there is actually a linting rule that will fail if you are missing a dependency that is used in the function because missing dependencies can have bad consequences in general.
See the lying section here https://overreacted.io/a-complete-guide-to-useeffect/.

But this is a good reminder that the linting rule is off so I'll re-add it and that will force us to fix it here.